TidepoolKit/Model/Datum/DeviceEventDatum/TAlarmDeviceEventDatum.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  TAlarmDeviceEventDatum.swift
+//  TidepoolKit
+//
+//  Created by Darin Krauss on 11/1/19.
+//  Copyright © 2019 Tidepool Project. All rights reserved.
+//
+
+import Foundation
+
+public class TAlarmDeviceEventDatum: TDeviceEventDatum, Decodable {
+    public enum AlarmType: String, Codable {
```

Comment:
Yes, *currently* `TAlarmDeviceEventDatum` is particularly focused on insulin delivery devices. That's primarily just because historically we haven't had other devices report specific alarms. However, with Tidepool Loop actively managing CGMs I can see we will probably want to expand the Tidepool data model to accommodate these new alarms/errors. We have yet to figure out what will need to be reported so these changes will come some time after this PR.